05-pvacview_tour.Rmd

```diff
@@ -105,21 +112,21 @@ The Ref Match column reflects whether or not the best peptide was found in the
 reference proteome which is undesired since such peptides are not novel and
 including them in a vaccine might lead to an auto immune response.
 
-Users are able to set a status for each candidate in the Evaluation column to mark
-them as Accept, Reject, or requiring further review.
+Users are able to set a status for each candidate by clicking the appropriate
+buttons: thumbs-up (Accept), thumbs-down (Reject), or Flag (requires further review).
```

Comment:
I'd prefer if the capitalization of the buttons and the parentheticals matched for all three buttons.